crates/provider-http-server/README.md

```diff
@@ -1,20 +1,38 @@
 
-For more information on the operations supported by this provider, please check out its corresponding [interface](https://github.com/wasmCloud/interfaces/blob/main/httpserver/httpserver.smithy).
+## Building
 
-Run `make` to compile to a native executable and build the par file.
-The par file is created in `build/httpserver.par.gz`.
+This HTTP server can be built from the root of this repository with `wash build`.
 
-## Link Definition Configuration Settings
-Configuration settings for the httpserver provider are described in [settings](./settings.md). 
+```shell
+wash build -p src/bin/http-server-provider
+```
 
-The default listen address is 127.0.0.1 port 8000.
+## Provider Configuration
 
-### ⚠️ Caution - Port Ownership
-If the instance of this capability provider running on a single host is linked to multiple components attempting to claim the same port, only the first **link definition** for that port will succeed, and the subsequent attempts will fail. During development, 
-it is recommended to check ("tail") the wasmCloud host logs for success and error messages.
+The wasmCloud HTTP server has optional configuration that you can provide to it at startup. All configuration keys should be lowercased, as well as their values when applicable. See the [provider config](https://wasmcloud.com/docs/developer/providers/configure) documentation for information about defining and using this configuration.
 
-For more hands-on tutorials on building components, including HTTP server components,
-see the [wasmcloud.dev](https://wasmcloud.dev) website.
+| Key              | Value                  | Default        | Description                                                                                                                                                                                                            |
+| ---------------- | ---------------------- | -------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `routing_mode`   | "address","path"       | `address`      | Dictates the routing mode of the capability provider. `address` mode will listen on a new address for each component, and `path` mode will listen on a single address and route to components based on a provided path |
+| `listen_address` | A valid listen address | "0.0.0.0:8080" | Must be provided when using the `path` routing mode. The listen address to listen on and route to components based on path                                                                                             |
+
+<!-- TODO: consider setting this up as the default address? -->
+
```

Comment:
What is this TODO from?

Reply:
Oh dang, this is from a future enhancement that I left on accident. Will remove